draft-ietf-ohai-ohttp.md

```diff
@@ -1001,10 +1001,21 @@ A proxy MAY add information to requests if the client is aware of the nature of
 the information that could be added.  The client does not need to be aware of
 the exact value added for each request, but needs to know the range of possible
 values the proxy might use.  It is important to note that information added by
-the proxy can reduce the size of the anonymity set of clients at a server.
-
-A proxy can also generate responses, though it assumed to not be able to
-examine the content of a request (other than to observe the choice of key
+the proxy can reduce the size of the anonymity set of clients at a target server.
+
+Proxies MAY apply differential treatment to clients that engage in abusive
+behavior, e.g., by sending too many requests in comparison to other clients,
+or as a response to rate limits signalled from the target. Any such
+differential treatment can reveal information to the target that would not
+be revealed otherwise and therefore reduce the size of the anonymity set of
+clients at a target server. For example, if a proxy chooses to rate limit or
+block an abusive client, this means that any client requests which are not
+treated this way are known to be non-abusive by the target. Clients should
```

Comment:
The target can't tell which requests are tied to the same client. How would the target recognize that a client is being rate-limited or blocked?

Reply:
Got it. Yes, if the proxy only allows specific clients in that pass a certain bar of trust, then the target could know that the client is in that set of trusted clients.

I think the blocking is an easier example here — rate limiting doesn't actually tell you as clearly, I think.

Reply:
Yeah, true. Should we give an example with blocking to make this more clear?

Reply:
I'd suggest simply changing this: "proxy chooses to rate limit or block an abusive client" to "proxy chooses to block an abusive client"?

Reply:
Done.